spot_driver/spot_driver/spot_ros2.py

```diff
@@ -569,6 +569,9 @@ def __init__(self, parameter_list: Optional[typing.List[Parameter]] = None, **kw
             self.create_subscription(
                 JointState, "arm_joint_commands", self.arm_joint_cmd_callback, 100, callback_group=self.group
             )
+            self.create_subscription(
+                PoseStamped, "arm_pose_commands", self.arm_pose_cmd_callback, 100, callback_group=self.group
```

Comment:
Just out of curiosity do you want a queue size of a 100?

Reply:
Not necessarily.... I chose 100 because I wanted to give it a "reasonable bound" in terms of how fast we can blindly stream joint angles and have it execute as fast as possible. And I chose the same number for consistency for the cartesian pose streaming, but it makes sense to lower the pose streaming as compared to the joint angles. However, I haven't vigorously tested this yet with a full model and would be open to revisiting changing this number after using it in actual applications